.github/workflows/maven.yaml

```diff
@@ -7,20 +7,13 @@ on:
    
 jobs:
   build:
-    runs-on: ${{matrix.os}}
+    runs-on: ubuntu-latest
```

Comment:
Only Linux for push (PR are handled in a separate workflow pr-windows-macos.yaml